### ISSUE TITLE
TLDR page

### ISSUE THREAD

**Issue by igorshubovych** [yourfirstpr/yourfirstpr.github.io#35]
TLDR is a collection of simplified and community maintained man pages.

![TLDR example](http://tldr-pages.github.io/images/screenshot.png?s=200)

You can read more about TLDR on site:
http://tldr-pages.github.io/

Every man-page is just a Markdown file with command name, short description (1 sentence) and a few the most common examples. It is that easy. Check out several examples:

- [cd](https://github.com/tldr-pages/tldr/blob/master/pages/common/cd.md)
- [chown](https://github.com/tldr-pages/tldr/blob/master/pages/common/chown.md)
- [diff](https://github.com/tldr-pages/tldr/blob/master/pages/common/diff.md)
- [git-commit](https://github.com/tldr-pages/tldr/blob/master/pages/common/git-commit.md)

There are helpful advices in [Contributing Guidelines](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md).

To set the CLI up you need to run:
~~~
npm install -g tldr
tldr --update
~~~

Start contributing by creating a new one or fixing one of the existing pages.
